src/index.js

```diff
@@ -1,64 +1,67 @@
-require("dotenv").config();
-const { Octokit } = require("@octokit/core");
-const axios = require("axios").default;
-const CronJob = require("cron").CronJob;
+require('dotenv').config();
+const { Octokit } = require('@octokit/core');
+const axios = require('axios').default;
+const CronJob = require('cron').CronJob;
 
 const TOKEN = process.env.TOKEN;
 const COLUMN_NODE_ID = process.env.COLUMN_NODE_ID;
 const NOTIFIER_URL = process.env.NOTIFIER_URL;
 const MENTION = process.env.MENTION;
-const PR_TIME = process.env.PR_TIME || "0 9,18 * * 1-5";
-const SOZVON_TIME = process.env.SOZVON_TIME || "0 21 * * 1-5";
+const PR_TIME = process.env.PR_TIME;// || '0 9,18 * * 1-5';
```

Comment:
it is a good practise to set default values as it was here.

```suggestion
const PR_TIME = process.env.PR_TIME || '0 9,18 * * 1-5';
```

but if you want to remove it, then you should make it not-optional in `.env` file. then uncomment this line in `.env.example`

![image](https://user-images.githubusercontent.com/15259299/107027262-efd29500-67bc-11eb-9ef9-40491bb6ab50.png)

Reply:
Okay